src/Vendor/OCHamcrest/OCHamcrestIOS.framework/Versions/A/Headers/HCAllOf.h

```diff
@@ -0,0 +1,59 @@
+//  OCHamcrest by Jon Reid, http://qualitycoding.org/about/
+//  Copyright 2016 hamcrest.org. See LICENSE.txt
```

Comment:
Do we need to do something specific with the license when we vendor this library? It looks like the library uses the BSD license.

Would it make more sense to pull this in via submodule instead? Or import via CocoaPods?

Reply:
Will update our 3rd party notices as per what AppCenter does in their [thirdpartynotices.txt](https://github.com/microsoft/appcenter-sdk-apple/blob/develop/ThirdPartyNotices.txt)

Reply:
Updated to include all our third party libraries